### DISCUSSION TITLE
How to configure Nextjs13 to use less

### DISCUSSION THREAD

**Discussion by melva10** [vercel/next.js#43557] [Help]
### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

。

### Which example does this report relate to?

。

### What browser are you using? (if relevant)

。

### How are you deploying your application? (if relevant)

。

### Describe the Bug

。

### Expected Behavior

。

### To Reproduce

。

**Comment by hqwlkj** [comment]
I also want to know

**Reply by haiifeng** [reply]
me too.
